poc/tenant-controller/pkg/controllers/tenants/controller.go

```diff
@@ -14,12 +14,14 @@ package tenants
 import (
 	"context"
 	"fmt"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
could you move this import to next group? (away from standard packages)

Reply:
okay, will do. I had it in the next group originally but was moved up automatically by my vim-go plugin's built-in formatter, but will move it back now in any case.